src/lib/helpers/viewerStyles.js

```diff
@@ -0,0 +1,22 @@
+export const viewerStyles = {
```

Comment:
this can be renamed to jsonViewerDefaultStyles and the other styles to jsonEditorDefaultStyles. The styles can be overriden and these are just defaults

Reply:
Resolved by renaming the files to jsonViewerDefaultStyles and jsonViewerDefaultStyles